tests/unit/ListShareClass.spec.ts

```diff
@@ -0,0 +1,156 @@
+// Libraries
+import Vue from 'vue'
+import Vuelidate from 'vuelidate'
+import Vuetify from 'vuetify'
+import flushPromises from 'flush-promises'
+
+// Utils
+import { createLocalVue, mount } from '@vue/test-utils'
+
+// Components
+import { ListShareClass } from '@/components/CreateShareStructure'
+
+Vue.use(Vuetify)
+Vue.use(Vuelidate)
+
+describe('List Shares and Series', () => {
+  let vuetify
+  let wrapper
+
+  const shareClasses = [
+    {
+      id: 1,
+      name: 'Common Shares',
+      priority: 0,
+      maxNumberOfShares: 10000,
+      parValue: 1.00,
+      currency: 'Canadian Dollar ($CDN)',
```

Comment:
Should this match what the currency list (mixin) provides?

Reply:
Yes. it must be the code. Fixed